src/main/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,129 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * class movie data base.
+ */
+public class MovieDatabase {
+    private List<Movie> movieList;
+    private List<Actor> actorList;
+
+    /**
+     * Initializes the MovieDatabase object.
+     */
+    public MovieDatabase() {
+        this.movieList = new ArrayList<>();
+        this.actorList = new ArrayList<>();
+    }
+
+   /**
+     * Getter for the movies in the database.
+     * @return  all the movies in the bd
+     */
+    public List<Movie> getMovieList() {
+        return this.movieList;
+    }
+
+    /**
+     * Getter for the actors in the database.
+     * @return an ArrayList of all the actors in the
+     * database
+     */
+    public List<Actor> getActorList() {
+        return this.actorList;
+    }
+
+    /**
+     * Getter for the String names of the movies.
+     * @return ArrayList of Strings of the names of all
+     * the movies in the database.
+     */
+    private ArrayList<String> getMovieNames() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getMovieNames" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)